ksonnet/service/etcd.libsonnet

```diff
@@ -0,0 +1,46 @@
+{
+    all(params):: [
+    {
+	"apiVersion": "v1",
+	    "kind": "Service",
+	    "metadata": {
+		"name": "dkube-etcd-server",
+		"namespace": "dkube"
+	    },
+	    "spec": {
+		"ports": [
+		{
+		    "port": 2379,
+		    "protocol": "TCP",
+		    "targetPort": 2379
+		}
+		],
+		"selector": {
+		    "app": "dkube-etcd-server"
+		},
+		"type": "ClusterIP"
+	    }
+    },
+    {
+	"kind": "PersistentVolumeClaim",
+	"apiVersion": "v1",
+	"metadata": {
+	    "name": "etcd-pvc",
+	    "namespace": "dkube"
+	},
+	"spec": {
+	    "accessModes": [
+		"ReadWriteMany"
+	    ],
+	    "storageClassName": "",
+	    "resources": {
+		"requests": {
+		    "storage": "100Gi"
+		}
+	    },
+	    "volumeName": params.etcdStoragePV
```

Comment:
who is passing this?

Reply:
we are setting params in prototype/all.jasonnet file. If parameter's value is not passed then it will take default value